### ISSUE TITLE
Error: Cannot find module 'eslint-config-standard-jsx' from '/Users/kvz/code/frey'

### ISSUE THREAD

**Issue by kvz** [feross/standard#407]
Hey there,

Thanks for a great project!

I think I may have found an issue just now.

When using `6.0.1` with `eslint-babel` and running `standard .` I'm getting:

```bash
standard: Unexpected linter output:

Error: Cannot find module 'eslint-config-standard-jsx' from '/Users/kvz/code/frey'
Referenced from: /usr/local/lib/node_modules/standard/.eslintrc
    at Function.module.exports [as sync] (/usr/local/lib/node_modules/standard/node_modules/eslint/node_modules/resolve/lib/sync.js:33:11)
    at resolve (/usr/local/lib/node_modules/standard/node_modules/eslint/lib/config/config-file.js:393:38)
    at load (/usr/local/lib/node_modules/standard/node_modules/eslint/lib/config/config-file.js:412:24)
    at /usr/local/lib/node_modules/standard/node_modules/eslint/lib/config/config-file.js:331:36
    at Array.reduceRight (native)
    at applyExtends (/usr/local/lib/node_modules/standard/node_modules/eslint/lib/config/config-file.js:314:28)
    at Object.load (/usr/local/lib/node_modules/standard/node_modules/eslint/lib/config/config-file.js:436:22)
    at loadConfig (/usr/local/lib/node_modules/standard/node_modules/eslint/lib/config.js:67:33)
    at new Config (/usr/local/lib/node_modules/standard/node_modules/eslint/lib/config.js:192:38)
    at CLIEngine.executeOnFiles (/usr/local/lib/node_modules/standard/node_modules/eslint/lib/cli-engine.js:492:28)

If you think this is a bug in `standard`, open an issue: https://github.com/feross/standard/issues
```

I switched from using eslint, so I removed `node_modules` and ran `npm install` to make sure this wasn't the culprit. The same error happened.

When switching to the older standard `5.4.1` running `standard .`, it's all good.

This is on OSX with node v4.2.1 for what it's worth

**Comment by tlvince** [comment]
See #406.

**Comment by feross** [comment]
This should be fixed in `standard` 6.0.3. Sorry for the trouble!
